asb-native/src/main/java/org/ballerinax/asb/util/ASBUtils.java

```diff
@@ -0,0 +1,145 @@
+package org.ballerinax.asb.util;
```

Comment:
Missing licence header

Reply:
Added licence header.
/Thanks

Reply:
Can't see the updated license.

Reply:
Sorry. I have mistakenly done the changes on another branch. 
Now I corrected. Please check now.